libs/components/src/lib/JobUpdateForm/index.tsx

```diff
@@ -0,0 +1,189 @@
+import { Fragment, useEffect, useState } from 'react';
+import { Form, Space, Typography } from 'antd';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import {
+  description,
+  GetJobResponse,
+  hourly_rate,
+  jobUpdateSchema,
+  JobUpdateValues,
+  routes,
+  StyledButton,
+  StyledInput,
+  StyledInputNumber,
+  StyledTextArea,
+  title,
+} from '@freelance/components';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  useGetJobQuery,
+  useUpdateJobMutation,
+} from 'src/redux/job-post/job-post';
+
+import {
+  ButtonWrapper,
+  FormWrapper,
+  PropertiesWrapper,
+  StyledErrorMessage,
+  TitleWrapper,
+} from './styles';
+
+const { Title, Text } = Typography;
+
+export const JobUpdateForm = () => {
+  const [job, setJob] = useState<GetJobResponse>();
+  const { t } = useTranslation();
+
+  const {
+    control: jobUpdateControl,
+    handleSubmit: jobUpdateHandleSubmit,
+    formState: { errors },
+  } = useForm<JobUpdateValues>({
+    resolver: yupResolver(jobUpdateSchema),
+  });
+
+  const navigate = useNavigate();
+
+  const temporallyMagicNumber = 48;
```

Comment:
if its temporary setup above `//TODO temporary should be changed when we implement chat feature` TODO word used for easily find all tech debs, in other case you will forget

Reply:
fixed